frontend/src/pages/collab.js

```diff
@@ -261,8 +272,8 @@ function Collab() {
           </div>
           <div className="collab-question-content">
             <b className="question-title">{storedQuestion.id}. {storedQuestion.title}</b>
-            <Tooltip title="Leave collaboration room">
-              <Button onClick={handleLeaveRoom} style={{marginBottom: '1%'}}>
+            <Tooltip title="Leave&Save">
```

Comment:
I think its ok to add a bit more explanation here like "Save code and leave collaboration room"